delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/files/Files.scala

```diff
@@ -197,6 +197,39 @@ final class Files(
     } yield res
   }.span("createLink")
 
+  // TODO comments
+  def copy(
+      id: FileId,
+      targetProject: ProjectRef,
+      targetStorageId: Option[IdSegment],
+      tagForCopy: Option[UserTag]
```

Comment:
The user can optionally provide a id for the file or a new one can be generated.
One of the reasons behind that is now most of the time, the org/project appears in the id so that an id is unique in a deployment so keeping the id would break that.

Reply:
Right so something like `destinationFileId: Option[IdSegment]` passed into the endpoint?